### PR TITLE
Add rubygem-rbnacl package

### PR DESCRIPTION
**Pull request by adamruzicka**
Different versions of net-ssh have different dependencies for ed25519 key
support. Net-ssh 4.2 line needs rbnacl and rbnacl-libsodium.

- Add rubygem-rbnacl package
- Add rubygem-rbnacl-libsodium package

CPs:
- [ ] 2.5

### CODE REVIEW

**Comment by evgeni** [comment]
this needs to be in two PRs, as rbnacl-libsodium depends on rbnacl and this won't be found until the first part is merged and built

File: comps/comps-foreman-plugins-rhel7.xml [hunk 1]
```diff
@@ -133,6 +133,7 @@
       <packagereq type="default">tfm-rubygem-puppetdb_foreman</packagereq>
       <packagereq type="default">tfm-rubygem-radcli</packagereq>
       <packagereq type="default">tfm-rubygem-rainbow</packagereq>
+      <packagereq type="default">tfm-rubygem-rbnacl</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-rhel7.xml:136]: this also needs to go to `comps-foreman-plugins-el8.xml` without the tfm prefix

File: rel-eng/tito.props [hunk 2]
```diff
@@ -914,6 +914,7 @@ whitelist = foreman-discovery-image-service
   rubygem-puppetdb_foreman
   rubygem-radcli
   rubygem-rainbow
+  rubygem-rbnacl
```
Comment (evgeni) [inline rel-eng/tito.props:917]: this needs to be added to the el8 plugins group too

**Comment by evgeni** [comment]
```console
% koji add-pkg foreman-plugins-nightly-rhel7 tfm-rubygem-rbnacl tfm-rubygem-rbnacl-libsodium --owner=kojiadm
Adding 2 packages to tag foreman-plugins-nightly-rhel7
% koji add-pkg foreman-plugins-nightly-el8 rubygem-rbnacl rubygem-rbnacl-libsodium --owner=kojiadm
Adding 2 packages to tag foreman-plugins-nightly-el8
```

File: comps/comps-foreman-plugins-rhel7.xml (hunk 1 above)
Comment (adamruzicka) [reply comps/comps-foreman-plugins-rhel7.xml:136]: changed

File: packages/plugins/rubygem-rbnacl/rubygem-rbnacl.spec [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/.coveralls.yml
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.rubocop.yml
+%{gem_instdir}/.ruby-version
```
Comment (evgeni) [inline packages/plugins/rubygem-rbnacl/rubygem-rbnacl.spec:76]:
```suggestion
%exclude %{gem_instdir}/.ruby-version
```

File: packages/plugins/rubygem-rbnacl/rubygem-rbnacl.spec [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/.coveralls.yml
```
Comment (evgeni) [inline packages/plugins/rubygem-rbnacl/rubygem-rbnacl.spec:73]:
```suggestion
%exclude %{gem_instdir}/.coveralls.yml
```

**Review by evgeni — CHANGES_REQUESTED** [review]
I *think* that's the last round of "and theeeeeeeeeeeeen?" (sorry @pcreech, I am totally stealing this one)

**Review by evgeni — APPROVED** [review]

**Pull request merged**